saddle-ops-inlined/src/test/scala/org/saddle/macros/MatAbritraries.scala

```diff
@@ -0,0 +1,39 @@
+/** Copyright (c) 2013 Saddle Development Team
```

Comment:
Were these files copied? Why?

Reply:
using "test->test" dependency produced an error.  The problem was that it put both the 2.13 and the 3 artifacts on the class path. Not using "test->test" submodule dependencies cleared it up.

Looked like an unwanted behavior in sbt around the interaction of the "test->test" feature and the crossScalaVersions feature.

Reply:
I'd like to look further into this one, prevent code duplication.